.travis.yml

```diff
@@ -11,6 +15,8 @@ addons:
     packages:
       - mysql-server
   chrome: stable
+
+# Default steps within each job
 before_install:
```

Comment:
We can modify these steps to only install what is needed for each test separately, I'm just not sure which steps a re relevant to e2e and unit